mockito-core/build.gradle.kts

```diff
@@ -73,6 +78,24 @@ tasks {
         into(generatedInlineResource.map { it.dir("org/mockito/internal/creation/bytebuddy/inject") })
 
         rename("(.+)\\.class", "$1.raw")
+
+        /*TODO: How to do this?
```

Comment:
If I understand correctly, you are curious how to hook this up with Gradle? If that's the case, then you should wrap this in a `doLast` block similar to what we do in `createTestResources`. In that case, the code doesn't run during configuration, but it runs in the execution phase of Gradle.

Reply:
Ok, I gave this several attempts, but I have still not become friends with Gradle. How do I get hold if the `module-info.class` file in `doLast`? All I want, is to get hold of that file, and transform it.

Reply:
I managed, but there is another obstacle. There must not be a jar entry for a folder in *inject*, as this triggers the class loader to reserve the package name and avoid the boot loader. I thought I adjusted this, by moving up the raw class file one level, but now I am stuck again as it fails me on some obscure jar bundling error. If any Gradle user knows how to resolve this, I'd be thankful for the hint. Otherwise, I will try again next week.